src/middlewares/auth.middleware.js

```diff
@@ -0,0 +1,41 @@
+const jwt = require("../lib/jwt.lib");
+const createError = require("http-errors");
+const { getUser } = require("../usecases/user.usecase");
+
+const auth = (request, response, next) => {
+  try {
+    const authorization = request.headers.authorization || "";
+    const token = authorization.replace("Bearer ", "");
+    const verifiedToken = jwt.verify(token);

```

Comment:
la constante verifiedToken es utilizada?. Si no es asi, la puedes ahorrar, con utilizar el .verify de permite o te lanza errores si el token es correcto 
```suggestion
    jwt.verify(token);

```

Reply:
de acuerdo con esta, no se esta usando